templates/blocks/featured_record_article.html

```diff
@@ -0,0 +1,37 @@
+{% load static wagtailimages_tags wagtailcore_tags %}
+
+{% if value.page.live or request.is_preview %}
```

Comment:
Thought: It's strange to handle this in the FE and could lead to a broken experience for editors that would be hard to figure out. I'm not suggestion that this is not the correct way but feel's fragile, is there anything we can do in the BE to enforce this?

Reply:
Problem there is... it's perfectly valid for in-progress content to link to other in-progress content (maybe they are working on a 'section' of related content). Pages also change states in a way that is difficult to enforce, and probably shouldn't be enforced. If you linked to a live page, then that page is later unpublished temporarily while it undergoes an update, does that mean the page with the link is now invalid?

Reply:
I can tell you from experience that 'validation' isn't the answer :)

Reply:
There might be some way to do this sort of thing with custom chooser and structblocks in Wagtail, but that would definitely need to be a separate piece of work.